src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -665,7 +665,7 @@ App::DocumentObjectExecReturn *Cone::execute()
             BRepPrimAPI_MakeCone mkCone(Radius1.getValue(),
                                         Radius2.getValue(),
                                         Height.getValue(),
-                                        Angle.getValue()/180.0f*M_PI);
+                                        Base::toRadians<double>(angle));

```

Comment:
@yorikvanhavre CI is failing due to an error in solving a merge conflict here.
This line should be:
```c++
                                        Base::toRadians<double>(Angle.getValue()));
```

Reply:
Thanks! Indeed that looked wrong :sweat_smile: